lib/features/modules_carousel_element.dart

```diff
@@ -20,7 +31,7 @@ class ModulesCarouselElement extends StatelessWidget {
       },
       child: Container(
         height: 115,
-        constraints: const BoxConstraints(minWidth: 155, maxWidth: 175),
+        width: module.title == "Réanimation Cardio-pulmonaire" ? 178 : 155,
```

Comment:
not a good way to handle it

Reply:
An other idea ? Because I searched a long time to make the size depend on the text but it creates a lot of issues